client/src/api/util.js

```diff
@@ -7,13 +7,24 @@ export const relog = () => {
 }
 
 export const request = (method, endpoint, data) => {
-  return fetch(config.apiEndpoint + endpoint, {
+  let body = null
+  let qs = ''
+  if (method === 'GET' && data) {
```

Comment:
Why not do `qs='?'` here and replace "fetch(`${config.apiEndpoint}${endpoint}?${qs}`" with "fetch(`${config.apiEndpoint}${endpoint}${qs}`". This way we don't have the extra "?"